tests/test_api.py

```diff
@@ -35,7 +35,8 @@ def test_veracode_xml_api_version(self):
         """
         Test the VeracodeXMLAPI version property
         """
-        veracode_xml_api = VeracodeXMLAPI()
+        with patch("veracode.api.get_app_id", return_value="1337"):
```

Comment:
Please do this in the 90 bazillion places across the various tests.
```suggestion
        with patch("veracode.api.get_app_id", return_value=constants.VALID_UPLOAD_API["app_id"]):
```